api/coreservice.go

```diff
@@ -132,6 +132,8 @@ type (
 		EVMNetworkID() uint32
 		// ChainID returns the chain id of evm
 		ChainID() uint32
+		// IsDeployerWhitelisted returns if the replay deployer is whitelisted
+		IsDeployerWhitelisted(address.Address) bool
```

Comment:
for check the received action

Reply:
unnecessary to do filtering inside API

Reply:
i think we should? otherwise it enters workingset and throws error
similar to ChainID check, we filter and reject inside API as well (also check and throw error in workingset)